cpp/include/ucxx/log.h

```diff
@@ -69,11 +69,12 @@ const std::unordered_map<std::string, ucxx_log_level_t> logLevelNames = {
   {"INFO", UCXX_LOG_LEVEL_INFO},
   {"DEBUG", UCXX_LOG_LEVEL_DEBUG},
   {"TRACE", UCXX_LOG_LEVEL_TRACE},
-  {"TRACE_REQ", UCXX_LOG_LEVEL_TRACE_REQ},
-  {"TRACE_DATA", UCXX_LOG_LEVEL_TRACE_DATA},
-  {"TRACE_ASYNC", UCXX_LOG_LEVEL_TRACE_ASYNC},
-  {"TRACE_FUNC", UCXX_LOG_LEVEL_TRACE_FUNC},
-  {"TRACE_POLL", UCXX_LOG_LEVEL_TRACE_POLL},
+  {"REQ", UCXX_LOG_LEVEL_TRACE_REQ},
+  {"DATA", UCXX_LOG_LEVEL_TRACE_DATA},
+  {"ASYNC", UCXX_LOG_LEVEL_TRACE_ASYNC},
+  {"FUNC", UCXX_LOG_LEVEL_TRACE_FUNC},
+  {"POLL", UCXX_LOG_LEVEL_TRACE_POLL},
+  {"", UCXX_LOG_LEVEL_LAST},
   {"PRINT", UCXX_LOG_LEVEL_PRINT}};
```

Comment:
Should these be in the opposite order?

Reply:
No, the [print level](https://github.com/openucx/ucx/blob/662455a15c4befc26b2baee45f1fc3b9cd6661b1/src/ucs/debug/log.c#L75) is a sort of "temporary debug", which you can use for development debugging. I'm not even sure we can use it currently because [we're bound to UCX logger's internals checking for the highest level at compile-time](https://github.com/openucx/ucx/blob/662455a15c4befc26b2baee45f1fc3b9cd6661b1/src/ucs/debug/log.c#L288-L291), but I'm leaving it here for the time being as I would like to fix that in the future.